docs/how_to/install.rst

```diff
@@ -1,671 +1,519 @@
+.. _installation:
+
 #############################
 Installing django CMS by hand
 #############################
 
-This is how to install django CMS 'the hard way' (it's not really that hard, but there is an easier
-way).
-
-It's suitable if you want to dive in to integrating django CMS into an existing project, are already
-experienced at setting up Django projects or indeed like to do things the hard way.
-
-If you prefer an easier way using an automated configuration tool - definitely recommended for new
-users - see :doc:`/introduction/install`, which is part of a complete introductory tutorial.
+The easiest way to install django CMS is by using the automated `django CMS installer
+<https://github.com/nephila/djangocms-installer>`_. This is the recommended way to start with new projects, and it's
+what we use in the :ref:`tutorial section of this documentation <tutorials>`.
 
-This document assumes you are familiar with Python and Django. After you've
-integrated django CMS into your project, you should be able to follow the
-:doc:`/introduction/index`.
+If you prefer to do things manually, this how-to guide will take you through the process.
 
-.. _requirements:
+..  note::
 
-************
-Requirements
-************
+    You can also use this guide to help you install django CMS as part of an existing project. However, the guide
+    assumes that you are starting with a blank project, so you will need to adapt the steps below appropriately as
+    required.
 
-* `Python`_ 2.7, 3.3, 3.4 or 3.5.
-* `Django`_ 1.8.x, 1.9.x, 1.10.x
-* `django-classy-tags`_ 0.7.0 or higher
-* `django-treebeard`_ 4.x
-* `django-sekizai`_ 0.8.2 or higher
-* `djangocms-admin-style`_ 1.0 or higher
-* An installed and working instance of one of the databases listed in the
-  `Databases`_ section.
+This document assumes you have some basic familiarity with Python and Django. After you've integrated django CMS into
+your project, you should be able to follow the :doc:`/introduction/index` for an introduction to developing with django
+CMS.
 
-.. note:: When installing the django CMS using pip, all of the dependencies
-          will be installed automatically.
 
-.. _Python: https://www.python.org
-.. _Django: https://www.djangoproject.com
-.. _django-classy-tags: https://github.com/ojii/django-classy-tags
-.. _django-treebeard: http://code.tabo.pe/django-treebeard/src
-.. _django-sekizai: https://github.com/ojii/django-sekizai
-.. _djangocms-admin-style: https://github.com/divio/djangocms-admin-style
+******************************
+Install the django CMS package
+******************************
 
-Recommended
-===========
+Check the :ref:`Python/Django requirements <requirements>` for this version of django CMS.
 
-These packages are not *required*, but they provide useful functionality with
-minimal additional configuration and are well-proven.
+django CMS also has other requirements, which it lists as dependencies in its ``setup.py``.
```

Comment:
Then shouldn't we just link to the setup.py and not have a `Python/Django requirements <requirements>` section